build.gradle

```diff
@@ -55,15 +54,11 @@ compileJava   {
     targetCompatibility = 1.8
 }
 
-configurations.all {
-    // check for updates every build
-    resolutionStrategy.cacheChangingModulesFor 0, "seconds"
-}
-
```

Comment:
Why did you remove this?

Reply:
Re-added in b8c9f819de34c2ae7220db45d3e469e15d0a61c3.